src/StringUtil.php

```diff
@@ -6,6 +6,10 @@
 
 class StringUtil
 {
+    public const WINDOWS_NEW_LINE = "\r\n";
```

Comment:
```suggestion
    public const WINDOWS_NEWLINE = "\r\n";
```

Bei einer Suche im Projekt nach `"\r\n"` fallen weitere Wiederholungen dieses Wert auf.